specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/artifacts.json

```diff
@@ -203,6 +203,52 @@
         }
       }
     },
+    "/linkedservices/{linkedServiceName}/rename": {
+      "post": {
+        "tags": [
+          "linkedServices"
+        ],
+        "operationId": "LinkedService_RenameLinkedService",
+        "x-ms-examples": {
+          "LinkedServices_Rename": {
+            "$ref": "./examples/LinkedServices_Rename.json"
+          }
+        },
+        "description": "Renames a linked service.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/linkedServiceName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "request",
```

Comment:
If the use of this parameter is going to be the same everywhere (which it looks like it is) you might want to consider making a common parameter definition in `/parameters` (make sure you add `x-ms-parameter-location: method`).

Regardless of making it a common definition, you probably should add `"x-ms-client-flatten": true,` so that the parameter is flattened out of the containing object (since you've just got a single string)

Reply:
I am afraid to take this since it is used in other api's as well and don't want to introduce any bugs to existing api's. Is there anyway to raise a bug so that product team will work on the change.